src/components/buxy-table-transactions/buxy-table-transactions.js

```diff
@@ -0,0 +1,11 @@
+import template from "./buxy-table-transactions.html";
+
+export class buxyTableTransactionsComponent {
```

Comment:
class names should start from Capital letter

Reply:
please do not start component name from name of the application -- so it should be without 'buxy'